docs/guides/integration-prebid-client-side.md

```diff
@@ -41,20 +41,30 @@ An example of the UID2 Prebid.js client-side integration is available at the fol
 
 You'll need to complete the following steps:
 
-1. [Complete UID2 account setup](#complete-uid2-account-setup).
-2. [Add Prebid.js to your site](#add-prebidjs-to-your-site).
-3. [Configure the UID2 module](#configure-the-uid2-module).
+1. [Complete UID2 account setup and configure account](#complete-uid2-account-setup-and-configure-account)
+2. [Add Prebid.js to your site](#add-prebidjs-to-your-site)
+3. [Configure the UID2 module](#configure-the-uid2-module)
 
-### Complete UID2 Account Setup
+### Complete UID2 Account Setup and Configure Account
 
-Complete the UID2 account setup by following the steps described in the [Account Setup](../getting-started/gs-account-setup.md) page. As part of the account setup process for a client-side implementation, you'll need to provide a list of **domain names** for the sites that you'll be using with Prebid.js.
+To integrate with UID2, you'll need to have a UID2 account. If you haven't yet created an account, first follow the steps described on the [Account Setup](../getting-started/gs-account-setup.md) page.
+
+When initial account setup is complete, you'll receive instructions and a link to access the UID2 Portal, where you can create your [credentials](../getting-started/gs-credentials.md) for the production environment and configure additional values that you'll need to provide. For details, see [Getting Started with the UID2 Portal](../portal/portal-getting-started.md).
+
+For a client-side integration you'll need to set up these values, in the UID2 Portal on the [Client-Side Integration](../portal/client-side-integration.md) page:
+
+- Subscription ID and Public Key: See [Adding and Managing Key Pairs](../portal/client-side-integration.md#adding-and-managing-key-pairs)
+
+- A list of **domain names** for the sites on which you'll be using this SDK: See [Adding and Managing Root-Level Domains](../portal/client-side-integration.md#adding-and-managing-root-level-domains)
```

Comment:
maybe replace SDK with Prebid.js - may be a bit confusing to mention SDK suddenly